src/test/java/de/unistuttgart/iste/gits/courseservice/integration/QueryResourcesTest.java

```diff
@@ -35,17 +34,15 @@ void testResourceNotExisting(GraphQlTester tester){
         //GraphQL query
         String query = """
                 query {
-                    courseIdsByResource(ids: ["%s"]) {
-                        resource_id
-                        courses {
-                            course_id
-                            available
-                        }
+                    resourceById(ids: ["%s"]) {
+                        id
+                        availableCourses
+                        unAvailableCourses
```

Comment:
@myluki2000 do you think we need a list of "unavailable" course ids (i.e., the resource belongs to a course but the course is not started or public yet) -> I thought no, but Tim thought so, so maybe you have a better understanding than me?

Reply:
I'm not sure if we need it, so I'd leave it in. "lieber haben und nicht brauchen als brauchen und nicht haben" as we say in German. Although I have 1 little nag: I think it should be named "unavailable" and not "unAvailable", as unavailable is one word.

Reply:
I mean the counter argument would be that you don't want to maintain things that you don't really need, but I'm fine with leaving it in.